MastermindLogic/src/test/java/nl/fhict/s3/mastermindlogic/MastermindLogicApplicationTests.java

```diff
@@ -247,49 +257,39 @@ void testInputColour() {
 				EPinColour.LIME,
 				EPinColour.BLUE
 		};
+		assertArrayEquals(ExpectColour, player1.getBoard().getCode(), "ERROR WRONG COLOUR INPUT");
```

Comment:
Waarom is ExpectColour niet expectColour?

Reply:
Geen idee. Waarom?